mixer/adapter/stackdriver/contextgraph/send.go

```diff
@@ -90,6 +90,33 @@ func (e edge) ToProto() *contextgraphpb.Relationship {
 	}
 }
 
+// splitBySize helps find the largest BatchRequest that is smaller than the max request size
+func splitBySizeEntity(msg *contextgraphpb.AssertBatchRequest) int {
+	curSize := 0
+	curIndex := 0
+	for ; curIndex < len(msg.EntityPresentAssertions); curIndex++ {
+		curSize += proto.Size(msg.EntityPresentAssertions[curIndex])
+		if curSize >= maxReq {
+			curIndex--
```

Comment:
It is not necessary to do `curIndex--` right? `allRequests[:split]` won't include element at index `split`.